packages/wrangler/src/__tests__/d1/execute.test.ts

```diff
@@ -6,6 +6,10 @@ import { runInTempDir } from "../helpers/run-in-tmp";
 import { runWrangler } from "../helpers/run-wrangler";
 import { writeWranglerConfig } from "../helpers/write-wrangler-config";
 
+// we want to include the banner to make sure it doesn't show up in the output
+// when --json=true
+vi.unmock("../../wrangler-banner");
```

Comment:
for posterity, this is mocked [here](https://github.com/cloudflare/workers-sdk/blob/82a89371250a1ec3e56fa5a518c685d362bde0ad/packages/wrangler/src/__tests__/vitest.setup.ts#L86)

Reply:
updated PR description to reflect this detail